scripts/genproto.sh

```diff
@@ -32,6 +32,7 @@ grpc_gateway_path=$(go list -m -f '{{.Dir}}' github.com/grpc-ecosystem/grpc-gate
 googleapis_path="$grpc_gateway_path/third_party/googleapis"
 
 echo "Generating protobuf for api/pb"
-compile -I. -I$googleapis_path --go_out=plugins=grpc:. api/pb/api.proto
-compile -I. -I$googleapis_path --grpc-gateway_out=logtostderr=true:. api/pb/api.proto
-compile -I. -I$googleapis_path --swagger_out=logtostderr=true:. api/pb/api.proto
+echo protoc -I. -Idevtools/include -I$googleapis_path --go_out=plugins=grpc:. api/pb/api.proto
```

Comment:
Wouldn't it be more useful to print:
```suggestion
echo $protoc -I. -Idevtools/include -I$googleapis_path --go_out=plugins=grpc:. api/pb/api.proto
```
(includes the full path that is actually run)

Reply:
This was for debugging purposes, didn't mean to leave it in! Thanks for catching this